custom/icds/views/custom_sms_report.py

```diff
@@ -22,22 +26,49 @@ def dispatch(self, *args, **kwargs):
 
     @property
     def page_context(self):
-        self.request_form = CustomSMSReportRequestForm()
-        return {
-            'request_form': self.request_form,
-        }
+        report_ids = self._get_active_report_ids()
+        report_count = len(report_ids)
+        context = {}
+        disable_submit = report_count >= MAX_CONCURRENT_SMS_REPORTS_ALLOWED
+        if disable_submit:
+            messages.error(self.request, _('Only {max_report_count} concurrent reports are allowed at a time.\
+                Please wait for them to finish.').format(
+                max_report_count=MAX_CONCURRENT_SMS_REPORTS_ALLOWED
+            ))
+        if report_count > 0:
+            context.update({
+                'reports_in_progress': ',  '.join(report_ids)
+            })
```

Comment:
nit: i dont think you need to use update there, just a normal assignment would do since the key would be present already for sure

Reply:
Yes. Updated the code.